nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -26,6 +26,14 @@
 | Region        | `level`       | `tree_depth` |
 | TenantGroup   | `level`       | `tree_depth` |
 
+### Renamed Foregin Key Fields
```

Comment:
Should we put these in the `Renamed Database Fields` table above?

Reply:
That was my original approach. My concern is that this approach will have separate 2.0 changes referencing the same table. Im not sure if it is important to make that distinction.

Reply:
IMO this belongs in `Renamed Database Fields` rather than adding a new/separate table.